alica_engine/include/engine/blackboard/Blackboard.h

```diff
@@ -105,7 +120,7 @@ class LockedBlackboardRW
     LockedBlackboardRW(LockedBlackboardRW&) = delete;
 
     template <class... Args>
-    [[deprecated]] void registerValue(const std::string& key, Args&&... args)
+    void registerValue(const std::string& key, Args&&... args)
```

Comment:
Dont undeprecate.  If anything we can remove this

Reply:
I have removed it in https://github.com/rapyuta-robotics/alica/pull/338/files